staging/src/github.com/openshift/oc/pkg/cli/admin/release/mirror.go

```diff
@@ -442,9 +452,57 @@ func (o *MirrorOptions) Run() error {
 	} else {
 		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
 	}
+
+	printInstallInstructions(o.Out, o.From, o.To, repoMap)
 	return nil
 }
 
+// printInstallInstructions provides detail to the user for using the new mirror for installs.
+// The instructions include yaml formatted output which should be added to the install-config.yaml.
+//
+// Exmaple output:
+//
+//  <descriptive installer usage text>
+//  imageContentSources:
+//  - sources:
+//    - mycompany.com/myrepository/repo                 # destination (mirror) listed first
+//    - registry.svc.ci.openshift.org/ocp/release       # source listed second
+//    - quay.io/openshift-release-dev/ocp-v4.0-art-dev  # repos from image-references
```

Comment:
@mtnbikenc 👋 

my 0.002$: in my humble opinion, would have been nice if the initial design took into account the separation of the source and target repo mirror in the same way `oc adm release mirror` has  `--from=` and `--to`

The yaml file could have had something like

```
imageContentSources:
//  - sources:
//     - registry.svc.ci.openshift.org/ocp/release       # source listed second
//     - quay.io/openshift-release-dev/ocp-v4.0-art-dev  # repos from image-references
//   - targets:
//     - mycompany.com/myrepository/repo                 # destination (mirror) listed first 

```
Doing so  i think everything would have been much more natural and align with the cli - source/target

Reply:
@DanyC97 The context of these instructions is for the installer, not the mirror command.  If thinking about the mirror command, sources and targets make sense.  For the installer, we are providing a list of sources where images could be pulled from.  In this case, they can be pulled from the mirror, the original location or potentially from some other repo which an image in the images-references file pointed to.  In the end, the order of these items makes no difference in the installer code, but we wanted a consistent UX for how these repos were presented.  Have a look at https://github.com/openshift/installer/pull/1910 also.

Reply:
thank you for taking the time to respond @mtnbikenc , i may have misunderstood the comment in the code `destination (mirror) listed first`.

is all clear now, thx